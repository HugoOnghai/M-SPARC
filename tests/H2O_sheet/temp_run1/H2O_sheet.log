
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat H2O_sheet.inpt
# nprocs: 1
# Test: H2O sheet in x,y plane #
CELL: 5.67 5.67 24
MESH_SPACING: 0.2
FD_ORDER: 12
BOUNDARY_CONDITION: 3
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fd
SMEARING: 0.003674932
TOL_SCF: 1e-6
MIXING_VARIABLE: potential
MIXING_PRECOND: none
NSTATES: 8
PRINT_FORCES: 1
PRINT_ATOMS: 1

CALC_STRESS: 1
<\INPT>

Warning: "BOUNDARY_CONDITION" is obsolete, use "BC" instead!
 Reading .ion file ...
 Number of atom types : 2
 Total number of atoms: 3
 Default atomic mass for H is 1.007975
 Default atomic mass for O is 15.999400
 pseudo_dir # 1 = H_GGA.psp8
 pseudo_dir # 2 = O_GGA.psp8

 COORD:
   4.265111864110533   2.834589187490385  12.556010134214192
   1.404066510870237   2.834589187490385  12.556010134214192
   2.834589187490385   2.834589187490385  11.448630624967949

   2.465935248830600

   2.422364114257600

 kpoint grid before symmetry:
     0     0     0

 kpoint grid after symmetry:
     0     0     0

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 35
## Poisson tolerance not provided, choosing poisson_tol ...
## poisson_tol is set to: 1.000e-08
## Pseudocharge tolerance not provided, choosing pseudocharge_tol ...
## pseudocharge_tol is set to: 1.000e-08
 Creating differentiation matrices ...
 Done. (1.286 sec)

 Estimated memory usage:
 Total:  288.05 MB
 orbitals            :   23.16 MB
 sparse matrices     :  244.63 MB
 global-size vectors :   10.13 MB
 mixing histories    :   10.13 MB

 Finding rb for H ...
 rb = {1.415 1.400 1.400}, int_b = -1.000103431296120, err_rb = 1.034e-04
 rb = {1.615 1.600 1.600}, int_b = -0.999991768506383, err_rb = 8.231e-06
 rb = {1.815 1.800 1.800}, int_b = -1.000011793549502, err_rb = 1.179e-05
 rb = {2.015 2.000 2.000}, int_b = -1.000009207408833, err_rb = 9.207e-06
 rb = {2.215 2.200 2.200}, int_b = -1.000008993149955, err_rb = 8.993e-06
 rb = {2.415 2.400 2.400}, int_b = -1.000008085646335, err_rb = 8.086e-06
 rb = {2.615 2.600 2.600}, int_b = -1.000006631450733, err_rb = 6.631e-06
 rb = {2.815 2.800 2.800}, int_b = -1.000004889493480, err_rb = 4.889e-06
 rb = {3.015 3.000 3.000}, int_b = -1.000003182377995, err_rb = 3.182e-06
 rb = {3.215 3.200 3.200}, int_b = -1.000001780220762, err_rb = 1.780e-06
 rb = {3.415 3.400 3.400}, int_b = -1.000000843344640, err_rb = 8.433e-07
 rb = {3.615 3.600 3.600}, int_b = -1.000000798458038, err_rb = 7.985e-07
 rb = {3.815 3.800 3.800}, int_b = -1.000001928283206, err_rb = 1.928e-06
 rb = {4.015 4.000 4.000}, int_b = -1.000003557352664, err_rb = 3.557e-06
 rb = {4.215 4.200 4.200}, int_b = -1.000006023999494, err_rb = 6.024e-06
 rb = {4.415 4.400 4.400}, int_b = -1.000008049252371, err_rb = 8.049e-06
 rb = {4.615 4.600 4.600}, int_b = -1.000008318642058, err_rb = 8.319e-06
 rb = {4.815 4.800 4.800}, int_b = -1.000008289704102, err_rb = 8.290e-06
 rb = {5.015 5.000 5.000}, int_b = -1.000009874059142, err_rb = 9.874e-06
 rb = {5.215 5.200 5.200}, int_b = -1.000009633691756, err_rb = 9.634e-06
 rb = {5.415 5.400 5.400}, int_b = -1.000009048551638, err_rb = 9.049e-06
 rb = {5.615 5.600 5.600}, int_b = -1.000009525590215, err_rb = 9.526e-06
 rb = {5.815 5.800 5.800}, int_b = -1.000010355495216, err_rb = 1.036e-05
 rb = {6.015 6.000 6.000}, int_b = -1.000003376800709, err_rb = 3.377e-06
 rb = {6.215 6.200 6.200}, int_b = -0.999999618444060, err_rb = 3.816e-07
 rb = {6.415 6.400 6.400}, int_b = -1.000000081434216, err_rb = 8.143e-08
 rb = {6.615 6.600 6.600}, int_b = -0.999999982171350, err_rb = 1.783e-08
 rb = {6.815 6.800 6.800}, int_b = -1.000000002828629, err_rb = 2.829e-09
 rb = {6.815 6.800 6.800}
 Finding rb for O ...
 rb = {1.820 1.800 1.800}, int_b = -6.000094832190644, err_rb = 9.483e-05
 rb = {2.020 2.000 2.000}, int_b = -6.000056433441955, err_rb = 5.643e-05
 rb = {2.220 2.200 2.200}, int_b = -6.000052540078705, err_rb = 5.254e-05
 rb = {2.420 2.400 2.400}, int_b = -6.000038587164902, err_rb = 3.859e-05
 rb = {2.620 2.600 2.600}, int_b = -6.000028146823936, err_rb = 2.815e-05
 rb = {2.820 2.800 2.800}, int_b = -6.000019826231701, err_rb = 1.983e-05
 rb = {3.020 3.000 3.000}, int_b = -6.000014162367410, err_rb = 1.416e-05
 rb = {3.220 3.200 3.200}, int_b = -6.000011012205508, err_rb = 1.101e-05
 rb = {3.420 3.400 3.400}, int_b = -6.000009870358689, err_rb = 9.870e-06
 rb = {3.620 3.600 3.600}, int_b = -6.000010758801439, err_rb = 1.076e-05
 rb = {3.820 3.800 3.800}, int_b = -6.000013595754494, err_rb = 1.360e-05
 rb = {4.020 4.000 4.000}, int_b = -6.000016860490278, err_rb = 1.686e-05
 rb = {4.220 4.200 4.200}, int_b = -6.000021014194117, err_rb = 2.101e-05
 rb = {4.420 4.400 4.400}, int_b = -6.000023708381680, err_rb = 2.371e-05
 rb = {4.620 4.600 4.600}, int_b = -6.000023013144562, err_rb = 2.301e-05
 rb = {4.820 4.800 4.800}, int_b = -6.000021667927570, err_rb = 2.167e-05
 rb = {5.020 5.000 5.000}, int_b = -6.000023156275061, err_rb = 2.316e-05
 rb = {5.220 5.200 5.200}, int_b = -6.000021415255194, err_rb = 2.142e-05
 rb = {5.420 5.400 5.400}, int_b = -6.000019123222988, err_rb = 1.912e-05
 rb = {5.620 5.600 5.600}, int_b = -6.000018845107937, err_rb = 1.885e-05
 rb = {5.820 5.800 5.800}, int_b = -6.000019309718517, err_rb = 1.931e-05
 rb = {6.020 6.000 6.000}, int_b = -6.000006077709917, err_rb = 6.078e-06
 rb = {6.220 6.200 6.200}, int_b = -5.999999318667283, err_rb = 6.813e-07
 rb = {6.420 6.400 6.400}, int_b = -6.000000145403481, err_rb = 1.454e-07
 rb = {6.620 6.600 6.600}, int_b = -5.999999968013573, err_rb = 3.199e-08
 rb = {6.820 6.800 6.800}, int_b = -6.000000005105552, err_rb = 5.106e-09
 rb = {6.820 6.800 6.800}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 8.000000015808

 Integration b_ref = 7.999999999999

 ****************************************
 *          Eself_ref = 67.834389       *
 ****************************************
 Done. (0.578605 s)
 Time for b calculation: 0.593 seconds.

 Starting calculating nonlocal projectors ...
 Done. (0.146555 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 9.9944e-09 in 293 iterations.
 Poisson problem took 5.465493s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 3.718373
 This SCF iteration took 3.078 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.312850
 This SCF iteration took 1.531 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = -0.190429
 This SCF iteration took 1.491 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = -0.225552
 AAR converged to a relative residual of 9.43821e-09 in 253 iterations.
 Poisson problem took 4.476108s
 Escc = 3.29524944
 Etot = -16.92082863
 Eatom = -5.64027621
 Error in SCF iteration: 5.3256e-01 
 This SCF iteration took 6.041 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = -0.212324
 AAR converged to a relative residual of 8.53645e-09 in 247 iterations.
 Poisson problem took 4.391039s
 Escc = 0.68904236
 Etot = -17.06253515
 Eatom = -5.68751172
 Error in SCF iteration: 9.9819e-02 
 This SCF iteration took 5.946 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = -0.196527
 AAR converged to a relative residual of 8.41833e-09 in 199 iterations.
 Poisson problem took 3.544591s
 Escc = 0.11441929
 Etot = -17.06619732
 Eatom = -5.68873244
 Error in SCF iteration: 3.0245e-02 
 This SCF iteration took 5.074 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = -0.189907
 AAR converged to a relative residual of 9.06579e-09 in 175 iterations.
 Poisson problem took 3.409073s
 Escc = 0.10456881
 Etot = -17.06631924
 Eatom = -5.68877308
 Error in SCF iteration: 3.4504e-02 
 This SCF iteration took 4.952 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = -0.186975
 AAR converged to a relative residual of 9.01525e-09 in 181 iterations.
 Poisson problem took 3.245225s
 Escc = -0.02222959
 Etot = -17.06655498
 Eatom = -5.68885166
 Error in SCF iteration: 5.9467e-03 
 This SCF iteration took 4.765 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = -0.188725
 AAR converged to a relative residual of 9.92385e-09 in 146 iterations.
 Poisson problem took 2.618786s
 Escc = 0.00855425
 Etot = -17.06656009
 Eatom = -5.68885336
 Error in SCF iteration: 2.0145e-03 
 This SCF iteration took 4.076 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = -0.188664
 AAR converged to a relative residual of 9.67073e-09 in 157 iterations.
 Poisson problem took 2.809845s
 Escc = -0.00686070
 Etot = -17.06656361
 Eatom = -5.68885454
 Error in SCF iteration: 2.7801e-03 
 This SCF iteration took 4.364 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = -0.188872
 AAR converged to a relative residual of 9.98913e-09 in 148 iterations.
 Poisson problem took 2.638592s
 Escc = -0.00035228
 Etot = -17.06656469
 Eatom = -5.68885490
 Error in SCF iteration: 3.2563e-04 
 This SCF iteration took 4.145 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = -0.188887
 AAR converged to a relative residual of 9.36897e-09 in 127 iterations.
 Poisson problem took 2.291633s
 Escc = 0.00080495
 Etot = -17.06656464
 Eatom = -5.68885488
 Error in SCF iteration: 3.5424e-04 
 This SCF iteration took 3.750 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = -0.188876
 AAR converged to a relative residual of 9.34512e-09 in 97 iterations.
 Poisson problem took 1.743763s
 Escc = 0.00014300
 Etot = -17.06656474
 Eatom = -5.68885491
 Error in SCF iteration: 3.5760e-05 
 This SCF iteration took 3.249 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = -0.188870
 AAR converged to a relative residual of 8.84166e-09 in 67 iterations.
 Poisson problem took 1.224036s
 Escc = -0.00000631
 Etot = -17.06656471
 Eatom = -5.68885490
 Error in SCF iteration: 5.8380e-06 
 This SCF iteration took 2.924 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = -0.188872
 AAR converged to a relative residual of 7.32685e-09 in 43 iterations.
 Poisson problem took 0.808452s
 Escc = 0.00000775
 Etot = -17.06656472
 Eatom = -5.68885491
 Error in SCF iteration: 2.6885e-06 
 This SCF iteration took 2.280 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 13 
 ========================= 
 Fermi energy = -0.188872
 AAR converged to a relative residual of 9.20074e-09 in 19 iterations.
 Poisson problem took 0.392769s
 Escc = 0.00000592
 Etot = -17.06656474
 Eatom = -5.68885491
 Error in SCF iteration: 1.0994e-06 
 This SCF iteration took 1.841 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 14 
 ========================= 
 Fermi energy = -0.188871
 AAR converged to a relative residual of 9.12755e-09 in 13 iterations.
 Poisson problem took 0.288479s
 Escc = 0.00000148
 Etot = -17.06656473
 Eatom = -5.68885491
 Error in SCF iteration: 4.0886e-07 
 This SCF iteration took 1.774 s.


 Finished SCF iteration in 14 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -17.066564734 Ha.       *
 *          Energy per atom = -5.688854911 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 0.890 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
  -0.026074353346689   0.000009856619996  -0.005682694213414
   0.026080901085900   0.000009864243080  -0.005683084963699
  -0.000006547739211  -0.000019720863076   0.011365779177113

 Max magnitude of forces (Ha/Bohr):   0.026692900806402


 Time for calculating forces: 1.394869 s.

["Stress in Ha/Bohr**2"


]  -0.006108558790914  -0.000000000950728
  -0.000000000950728  -0.005430506945235


["Stress in GPa (equivalent to all periodic system)"


]  -7.488333502937417  -0.000001165474004
  -0.000001165474004  -6.657126253155639


 Time for calculating stress: 3.894492 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  4.265112 	  2.834589 	 12.556010 
  1.404067 	  2.834589 	 12.556010 
  2.834589 	  2.834589 	 11.448631 

 Run-time of the program: 75.626659 seconds

ans = 

  struct with fields:

                  cell_typ: 1
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 1
                        L1: 5.670000000000000
                        L2: 5.670000000000000
                        L3: 24
                        Nx: 28
                        Ny: 28
                        Nz: 121
                         N: 94864
                      ecut: -1
              mesh_spacing: 0.200000000000000
                   kptgrid: [0 0 0]
                  kptshift: [0 0 0]
                      nkpt: [1 1 1]
                     tnkpt: 1
                      wkpt: 1
                        BC: 3
                       BCx: 0
                       BCy: 0
                       BCz: 1
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 8
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 1.160451743453262e+03
                       bet: 2.721138785697259e+02
                       npl: 35
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 2
                 Nelectron: 8
                     n_typ: 2
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 4.066525489385879e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 5.000000000000000e-06
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 0
            PrintForceFlag: 1
          PrintAtomPosFlag: 1
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'H2O_sheet'
            inputfile_path: '.'
                        XC: 'GGA_PBE'
                       Atm: [1x2 struct]
                     n_atm: 3
                     Atoms: [3x3 double]
           mvAtmConstraint: [3x3 double]
                    IsFrac: [2x1 double]
                    IsSpin: [2x1 double]
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [3x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.202500000000000
                        dy: 0.202500000000000
                        dz: 0.200000000000000
                        dV: 0.008201250000000
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [94864x1 double]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [364x1 logical]
                 isOutr_11: [364x1 logical]
                      I_11: [364x1 double]
                     II_11: [364x1 double]
                      V_11: [364x1 double]
                 isOutl_22: [364x1 logical]
                 isOutr_22: [364x1 logical]
                      I_22: [364x1 double]
                     II_22: [364x1 double]
                      V_22: [364x1 double]
                      I_33: [1531x1 double]
                     II_33: [1531x1 double]
                      V_33: [1531x1 double]
                 G_JOutl_1: [1138368x1 logical]
                 G_JOutr_1: [1138368x1 logical]
                     G_J_1: [1138368x1 double]
                     G_I_1: [1138368x1 double]
                     G_V_1: [1138368x1 double]
                 G_JOutl_2: [1138368x1 logical]
                 G_JOutr_2: [1138368x1 logical]
                     G_J_2: [1138368x1 double]
                     G_I_2: [1138368x1 double]
                     G_V_2: [1138368x1 double]
                     G_J_3: [1105440x1 double]
                     G_I_3: [1105440x1 double]
                     G_V_3: [1105440x1 double]
                   Lap_std: [94864x94864 double]
                    grad_1: [94864x94864 double]
                    grad_2: [94864x94864 double]
                    grad_3: [94864x94864 double]
                LapPreconL: [94864x94864 double]
                LapPreconU: [94864x94864 double]
              memory_usage: 302046984
               staticfname: 'H2O_sheet.static'
                 suffixNum: 0
                  outfname: 'H2O_sheet.out'
                  parallel: 0
                         b: [94864x1 double]
                     b_ref: [94864x1 double]
                    rho_at: [94864x1 double]
                     Eself: 36.211575336783987
                 Eself_ref: 67.834388892502901
                       V_c: [94864x1 double]
                 PosCharge: 8.000000015807956
                 NegCharge: -8.000000015807956
                    E_corr: -1.853925549539781e-04
                       rho: [94864x1 double]
                      Atom: [1x3 struct]
                       phi: [94864x1 double]
                      e_xc: [94864x1 double]
                 dvxcdgrho: [94864x1 double]
                       Vxc: [94864x1 double]
                      Veff: [94864x1 double]
                         X: [94864x7 double]
                         F: [94864x7 double]
          mixing_hist_fkm1: [94864x1 double]
          mixing_hist_xkm1: [94864x1 double]
                       psi: [94864x8 double]
    upper_bound_guess_vecs: [94864x1 double]
                    EigVal: [8x1 double]
                  lambda_f: -0.188871119259730
                       occ: [8x1 double]
                    Etotal: -17.066564733684693
                     Eband: -4.060857473623172
                       Exc: -4.274471334752198
                    Exc_dc: -5.541866551152521
                  Eelec_dc: 21.938656930079283
                      Eent: -1.553192479667513e-07
                      Escc: 1.478117061906988e-06
                   S_Debug: [1x1 struct]
                     force: [3x3 double]
                 abs_force: [3x1 double]
                    Stress: [2x2 double]
             stress_dimred: [2x3 double]

