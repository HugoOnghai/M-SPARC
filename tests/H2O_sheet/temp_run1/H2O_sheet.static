***************************************************************************
                            Atom positions                                 
***************************************************************************
Cartesian coordinates of atoms (Bohr):
      4.2651118641       2.8345891875      12.5560101342
      1.4040665109       2.8345891875      12.5560101342
      2.8345891875       2.8345891875      11.4486306250
Atomic forces (Ha/Bohr):
     -0.0260743533       0.0000098566      -0.0056826942
      0.0260809011       0.0000098642      -0.0056830850
     -0.0000065477      -0.0000197209       0.0113657792
Stress (Ha/Bohr**2)                :
 -6.1085587909E-03  -9.5072775102E-10 
 -9.5072775102E-10  -5.4305069452E-03 
Stress equiv. all periodic (GPa)   :
 -7.4883335029E+00  -1.1654740036E-06 
 -1.1654740036E-06  -6.6571262532E+00 
