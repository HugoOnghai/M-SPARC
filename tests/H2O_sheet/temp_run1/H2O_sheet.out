***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:08:39                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 5.670000 5.670000 24.000000 
FD_GRID: 28 28 120
FD_ORDER: 12
BC: P P D
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003674932
CHEB_DEGREE: 35
NSTATES: 8
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: H2O_sheet.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.202500 (Bohr)
Mesh spacing in y-direction        :  0.202500 (Bohr)
Mesh spacing in z direction        :  0.200000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  H2O_sheet.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  H_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.82 6.80 6.80
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  O 6
Pseudopotential                    :  O_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.82 6.80 6.80
Number of atoms of type 2          :  1
Estimated total memory usage       :  288.05 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.6402762099E+00        5.326E-01        12.138
2            -5.6875117183E+00        9.982E-02        5.934
3            -5.6887324393E+00        3.024E-02        5.072
4            -5.6887730789E+00        3.450E-02        4.950
5            -5.6888516606E+00        5.947E-03        4.762
6            -5.6888533619E+00        2.015E-03        4.074
7            -5.6888545375E+00        2.780E-03        4.362
8            -5.6888548977E+00        3.256E-04        4.142
9            -5.6888548792E+00        3.542E-04        3.748
10           -5.6888549121E+00        3.576E-05        3.247
11           -5.6888549039E+00        5.838E-06        2.922
12           -5.6888549082E+00        2.688E-06        2.279
13           -5.6888549141E+00        1.099E-06        1.839
14           -5.6888549112E+00        4.089E-07        1.774
Total number of SCF: 14    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.6888549112E+00 (Ha/atom)
Total free energy                  : -1.7066564734E+01 (Ha)
Band structure energy              : -4.0608574736E+00 (Ha)
Exchange correlation energy        : -4.2744713348E+00 (Ha)
Self and correction energy         : -3.6211760729E+01 (Ha)
Entropy*kb*T                       : -1.5531924797E-07 (Ha)
Fermi level                        : -1.8887111926E-01 (Ha)
Average force                      :  2.1581706331E-02 (Ha/Bohr)
Maximum force                      :  2.6692900806E-02 (Ha/Bohr)
Time for force calculation         :  1.395 (sec)
Max stress                         :  6.1085587909E-03 (Ha/Bohr**2)
Max stress equiv. all periodic     :  7.4883335029E+00 (GPa)
Time for stress calculation        :  3.897 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  75.627 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
